app/models/interest.rb

```diff
@@ -1,3 +1,5 @@
 class Interest < ApplicationRecord
-  belongs_to :user
+  self.inheritance_column = :_type_disabled
+  enum type: { health: 0, hobby: 1, work: 2 }
+  belongs_to :user, optional: true
```

Comment:
In rails 5.0 i can't create new User with assosiation Interested, i read it is some bug, I can 
do it in configuration new_framework_defaults like a 
Rails.application.config.active_record.belongs_to_required_by_default = false
or ad optional: true, or in User model inverse_of: :user 
But i chose optional :true